travis/sync.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/sh
+echo "Script is running"
+
+setup_git_user(){
+    git config --global user.email "travis@travis-ci.org"
+    git config --global user.name "travis"
+}
+
+set_git_remote(){
+    git remote -v
+    git remote remove origin
+    git remote add origin https://travis:${GH_TOKEN}@github.com/waveywaves/security-advisories.git 
+}
+
+sync_files(){
+    node tools/sync_up.js
+}
+
+commit_vuln_db(){
+    git checkout master
+    git add ecosystem
+    git add core
+    git commit --message "Travis build: $TRAVIS_BUILD_NUMBER"
+}
+
+push_changes(){
+    git push origin master
```

Comment:
Does that ends up with travis opening a PR or pushing on master on the main repo?

Reply:
Pushing to the main repo not opening to the PR

Reply:
I guess it's ok to push to master if we only need to sync it.

I assume we aren't expected to need to handle merge conflicts since we will only be moving it one way?